.travis.yml

```diff
@@ -54,3 +54,8 @@ deploy:
   project: build/package
   on:
     condition: $TRAVIS_OS_NAME = linux
+    
+cache:
+  directories:
```

Comment:
Sorry for just spotting this now. This won't do anything as the Android version does not get built (yet) via TravisCI.